package.json

```diff
@@ -46,7 +47,7 @@
     "tests:android:test:jacoco-report": "cd tests/android && ./gradlew jacocoAndroidTestReport",
     "tests:ios:build": "cd tests && yarn detox build --configuration ios.sim.debug",
     "tests:ios:build:release": "cd tests && yarn detox build --configuration ios.sim.release",
-    "tests:ios:manual": "cd tests && SIMCTL_CHILD_GULGeneratedClassDisposeDisabled=1 CC=clang CPLUSPLUS=clang++ LD=clang LDPLUSPLUS=clang++ yarn react-native run-ios",
+    "tests:ios:manual": "cd tests && SIMCTL_CHILD_GULGeneratedClassDisposeDisabled=1 CC=clang CPLUSPLUS=clang++ LD=clang LDPLUSPLUS=clang++ yarn react-native run-ios --simulator='iPhone 15 Pro'",
```

Comment:
This is not a common device definition and in fact doesn't appear in my default list of devices, so will fail for me.

Is there any reason to specify one here when any could do?
If we must specify can we use same definition as we use for e2e runs (currently iPhone 16 which may also be non-standard but at least we have to have it to run e2e, and that's driven by runner availability of device definitions...)

https://github.com/invertase/react-native-firebase/blob/b38802203086d30f5bb98e76892e5023e02b010e/tests/.detoxrc.js#L47-L53